### DISCUSSION TITLE
When using Bohrium, model_devi's "group_ size" is restricted

### DISCUSSION THREAD

**Discussion by Saberve** [deepmodeling/dpgen#1189] [General]
![image](https://user-images.githubusercontent.com/53855907/233096745-e7946313-7d70-4cb8-ae5c-d24f68a3ee60.png)
When I set the value of model_devi's group_size to 40 or 50,dpgen will report an error.

![image](https://user-images.githubusercontent.com/53855907/233098342-243df826-c25f-4cdc-a47d-27e6b6605cf3.png)
![image](https://user-images.githubusercontent.com/53855907/233099599-cc30c4e8-1cd6-49e4-915b-ad26d327bd74.png)
But when I set this value to 20, dpgen will work normally, but it will waste a lot of time compressing and uploading files as a result.

It seems that the data space given by the data type of this field in the database is too small

**Comment by njzjz** [comment]
Hi @KZHIWEI @ACoderHIT please take a look.
